docs/articles/vs-test-adapter/Adapter-Installation.md

```diff
@@ -30,10 +21,29 @@ To add it is a NuGet package, you must have an active solution, then follow thes
 4. Click 'Install'
 5. In the "Select Projects" dialog, you need to select at least one project to add the adapter to, see notes below.
 
+## Installing With the Extension Manager
+
+The use of the vsix extension manager is deprecated.  It still works, but at some time support for it will be removed from Visual Studio, and also newer versions of the adapter.
+
+We recommend you try to move to nuget based adapters.
+
+To install the NUnit Test Adapter using the Extension Manager, follow these steps:
+
+1. From within Visual Studio, select Tools | Extension Manager.
+2. In the left panel of the Extension Manager, select Online Extensions
+3. Locate (search for) the NUnit 3.0 Test Adapter in the center panel and highlight it.
+4. Click 'Download' and follow the instructions.
+
+Use the Extension Manager to ensure that the NUnit 3.0 Test Adapter is enabled.
+
 ## How to choose between Extension and NuGet package
 
-The Extension will apply to Visual Studio itself, and will work for all projects you use. All users of your solution need to install the Extension. If you use TFS Build you must also install the extension to the build system there.
+The use of the vsix extension manager is deprecated.  It still works, but at some time support for it will be removed from Visual Studio, and also newer versions of the adapter.
```

Comment:
```suggestion
The use of the VSIX extension manager is deprecated.  It still works, but at some time support for it will be removed from Visual Studio, and also newer versions of the adapter.
```